src/api.js

```diff
@@ -15,12 +15,34 @@ export async function getMovie(id) {
     return content.data;
 }
 
-export async function getReviews(query = '') {
+export async function getReviews(query = "") {
     const res = await fetch(APIData + "/reviews" + query);
     const content = await res.json();
     return content.data;
 }
 
+export async function postReview(review, verified = false) {
+    const res = await fetch(APIData + "/reviews", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify({ 
+            "data": {
+                "comment": review.comment,
+                "rating": review.rating,
+                "author": review.author,
+                "verified": verified,
+                "movie": review.movie,
+                "createdAt": new Date().toISOString(),
+                "updatedAt": new Date().toISOString(),  
```

Comment:
When using `Strapi`, these should not be necessary. The CMS will assign them values automatically.